src/helpers/helpers.dom.ts

```diff
@@ -105,20 +110,22 @@ function getCanvasPosition(
 
 /**
  * Gets an event's x, y coordinates, relative to the chart area
- * @param event
- * @param chart
```

Comment:
this was removed since it didn't provide any additional information than the function signature already provided (not even variable types).  If you still want this, I can revert this change.